src/WebJobs.Script.WebHost/Standby/StandbyManager.cs

```diff
@@ -109,6 +109,9 @@ internal static void ResetChangeToken()
             _standbyChangeToken = new CancellationChangeToken(_standbyCancellationTokenSource.Token);
         }
 
+        // for testing
+        protected virtual Timer CreateSpecializationTimer() => new Timer(OnSpecializationTimerTick, null, _specializationTimerInterval, _specializationTimerInterval);
```

Comment:
Might be cleaner to just allow _specializationTimerInterval to be overridden on construction via an optional ctor param. Then your test can just set it to Timeout.Infinite